packages/loader/container-loader/src/container.ts

```diff
@@ -2332,18 +2280,14 @@ export class Container
 	}
 
 	private processRemoteMessage(message: ISequencedDocumentMessage) {
-		// non-interactive clients will not have any pending state we want to save
-		if (this.offlineLoadEnabled && this.isInteractiveClient) {
-			this.savedOps.push(message);
-		}
 		const local = this.clientId === message.clientId;
 
 		// Allow the protocol handler to process the message
 		const result = this.protocolHandler.processMessage(message, local);
 
 		// Forward messages to the loaded runtime for processing
 		this.runtime.process(message, local);
-
+		this.containerStateManager.addSavedOp(message);
```

Comment:
why move this after process, rather than keep it before like it was?

Reply:
I noticed we added the message to the savedOps array before its being processed, which I assume could fail. I don't know what kind of bugs we could have by having an op in the savedOps array, but it failed afterwards so this ordering just reflects that a saved op was actually processed.